BUILD

```diff
@@ -28,6 +28,13 @@ config_setting(
     ],
 )
 
+config_setting(
+    name = "windows",
+    constraint_values = [
+        "@bazel_tools//platforms:windows",
```

Comment:
Does this encompass MinGW?

Reply:
Yes it does https://github.com/bazelbuild/bazel/blob/master/tools/cpp/BUILD.windows.tpl#L130